src/components/Navbar.tsx

```diff
@@ -37,25 +55,141 @@ const NavLink = ({ href, children }: { href: string; children: ReactNode }) => (
+                                Sign in now to register for classes
+                            </Text>
+                            <Button
+                                height="50px"
+                                width="100%"
+                                borderRadius="6px"
+                                background={colourTheme.colors.Blue}
+                                fontWeight="normal"
+                                fontSize="16px"
+                                color="white"
+                                mt="5"
+                                mb="15"
+                            >
+                                Sign In
+                            </Button>
+                            <Divider mt="15" mb="15" color="#cdcdcd" />
+                        </Box>
+                    )}
+                </DrawerHeader>
+                <DrawerBody>
+                    <Flex flexDirection="row" mt="5" mb="5" alignItems="center">
+                        <Icon as={MdOutlineDns} width={6} height={6} />
+                        <NavLink href="/">Browse Programs</NavLink>
+                    </Flex>
+                    <Flex flexDirection="row" mt="5" mb="5" alignItems="center">
+                        <Icon as={MdOutlineClass} width={6} height={6} />
+                        <NavLink href="/class">View My Classes</NavLink>
+                    </Flex>
+                    {loggedIn && (
+                        <Flex
+                            flexDirection="row"
+                            mt="5"
+                            mb="5"
+                            alignItems="center"
+                        >
+                            <Icon as={BiUserCircle} width={6} height={6} />
+                            <NavLink href="/myaccounts">Account</NavLink>
+                        </Flex>
+                    )}
+                </DrawerBody>
+                <DrawerFooter>
+                    {loggedIn && (
+                        <Box w="100%">
+                            <Divider mt="15" mb="15" />
+                            <Flex alignItems="center">
+                                <Icon as={MdLogout} width={6} height={6} />
+                                <Text
+                                    marginLeft="5"
+                                    cursor="pointer"
+                                    onClick={() => signOut()}
+                                >
+                                    Log Out
+                                </Text>
+                            </Flex>
+                        </Box>
+                    )}
+                </DrawerFooter>
+            </DrawerContent>
+        </Drawer>
+    );
+};
+
 export const DEFAULT_NAVBAR_HEIGHT = 16;
 
 export const Navbar: React.FC<NavbarProps> = (props) => {
     const router = useRouter();
+    const variant = useBreakpointValue({ base: "sidebar", lg: "navbar" });
```

Comment:
let's stick with booleans:
```suggestion
    const isSideBar = useBreakpointValue({ base: true, lg: false });
```